cypress/integration/hyva/catalog/category.spec.js

```diff
@@ -7,10 +7,7 @@ describe("Category page tests", () => {
     });
 
     it("Can visit the category page and filters on color red", () => {
-        cy.get(selectors.shopByColorFilter).contains('Color').click();
         cy.get(selectors.selectColorRed).click();
-
-        cy.get(selectors.shopByColorFilter).contains('Color').should("not.exist")
```

Comment:
This check is to confirm that an applied filter is no longer available as a filter to the visitor, as it is rendered under the "Active Filters"